### PR TITLE
[FrameworkBundle] Use PHP arrays over JSON strings

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | --
| License       | MIT

It is easier to maintain IMHO

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ApiAttributesTest.php [hunk 1]
```diff
@@ -815,33 +735,27 @@ public static function mapRequestPayloadProvider(): iterable
...
+            'content' => json_encode([
+                'comment' => 'Hello everyone!',
+                'approved' => false,
+            ]),
+            'expectedResponse' => json_encode([
+                'type' => 'https://tools.ietf.org/html/rfc2616#section-10',
+                'title' => 'An error occurred',
+                'status' => 400,
+                'detail' => 'Bad Request',
+            ]),
```
Comment (OskarStark) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ApiAttributesTest.php:758]: On my local machine, this test is now failing 🤔

Comment (MatTheCat) [reply src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ApiAttributesTest.php:758]: JSON is malformed in the test (trailing comma after `"approved": false`) so you cannot generate it using `json_encode`.

**Comment by derrabus** [comment]
We currently have JSON fixtures because we compare them to a JSON response we get from the application. That sounds perfectly reasonable to me. And about being easier to maintain: PhpStorm understands that the string is JSON and provides me with syntax highlighting and inspections.

I wouldn't change this code tbh.

**Comment by OskarStark** [comment]
Thanks for the feedback 👍

**Pull request closed without merging**